TuberculosisApp/app/src/main/java/com/rugged/tuberculosisapp/MainActivity.java

```diff
@@ -128,7 +138,15 @@ public boolean onOptionsItemSelected(MenuItem item) {
         }
 
         if (id == R.id.action_sign_out) {
-            //TODO: Sign out option
+            AccountManager am = AccountManager.get(this);
+            Account account = am.getAccounts()[0];
+            am.invalidateAuthToken(account.type, MainActivity.identification.getToken());
+            // TODO: Do we want to remove the account as well? If so uncomment..
```

Comment:
what happens if we remove the account?

Reply:
Then the account will no longer be in the account manager, ie if you go to settings>accounts it won't be there. Now it will be there, but the authToken connected to the account will be invalidated, so the user has to login again to get a new one which is then linked to the existing account. I think that if you for example log out of gmail it remembers that you have an account.